lib/src/simple_op_heads_store.rs

```diff
@@ -58,18 +58,7 @@ struct SimpleOpHeadsStoreLock {
 
 impl OpHeadsStoreLock for SimpleOpHeadsStoreLock {}
 
-impl OpHeadsStore for SimpleOpHeadsStore {
-    fn name(&self) -> &str {
-        Self::name()
-    }
-
-    fn update_op_heads(&self, old_ids: &[OperationId], new_id: &OperationId) {
-        self.add_op_head(new_id);
-        for old_id in old_ids {
-            self.remove_op_head(old_id)
-        }
-    }
-
+impl SimpleOpHeadsStore {
```

Comment:
nit: maybe better to merge in to the previous `impl` block?

Reply:
Oops, yes.